frontend/src/index.css

```diff
@@ -1,55 +1,150 @@
-.div_Gallery{
-  display: flex;
-  flex-direction: row;
-  background-image: url("public/cdc-lRud5FDezto-unsplash.jpg");
-  background-color: aliceblue ;
-  box-shadow: 6px 4px 8px;
-  border-radius: 12px;
-  justify-content: flex-end;
-  border: solid 1px grey;
-  margin: 20px;
-  padding: 10px;
-
-}
-.recipe-card{
-  display: flex;
-  flex-direction: column;
-  margin: 5px;
-  padding: 5px;
-  border: solid 5px grey;
-  border-radius: 5px;
-}
-.div_Header{
-  font-family: "fantasy", serif;
-  font-size: x-large;
-}
-.div_List{
+.recipe-gallery {
+    display: grid;
+    background-color: #ffffff;
+    grid-template-columns: repeat(auto-fit, minmax(100px, 1fr));
+    gap: 20px;
+    max-width: 800px;
+    margin: 0 auto;
+    padding: 20px;
+    border-radius: 18px;
+    text-decoration: none;
+}
+.recipe-card-container {
+    display: grid;
+    text-decoration: none;
+    grid-template-columns: repeat(3, 2fr);
+    gap: 5px;
+    max-width: 800px;
+    margin: 0 auto;
+    padding: 20px;
+}
+
+.recipe-card {
+    background-color: #c7c6c6;
+    color: rgb(66, 65, 65);
+    text-decoration: none;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+    border-radius:10px 25%;
+    border-style: revert;
+    overflow: hidden;
+}
+
+.recipe-card img {
+    width: 100%;
+    height: 150px;
+}
+
+.recipe-info {
+    padding: 15px;
+    text-decoration: none;
+}
+
+.recipe-title {
+    font-family: Chalkboard serif;
+    text-decoration: none;
+    color: rgb(66, 65, 65);
+    font-size: 20px;
+    font-weight: bolder;
+    margin: 0;
+}
+
+.recipe-description {
+    font-size: 14px;
+    margin: 10px 0;
+}
+
+.recipe-button {
+    background-color: #007BFF;
+    color: #fff;
+    padding: 5px 10px;
+    border: none;
+    border-radius: 5px;
+    cursor: pointer;
+}
+
+.recipe-button:hover {
+    background-color: #0056b3;
+}
+/* Stil für die Beschreibung (Textbereich) */
+textarea {
+    width: 100%;
+    height: 150px; /* Passe die Höhe nach Bedarf an */
+    padding: 5px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    font-size: 16px;
+    font-family: Menlo, sans-serif;
+}
+
+/* Stil für die Schritte und Kategorien */
+.flex-container {
     display: flex;
-    flex-direction: row;
-    font-family: Menlo, serif;
```

Comment:
Wenn diese Schriftart auf dem jeweiligen Rechner nicht existiert, gibt es eine Fallback auf die SystemUI. Ich weiß nicht, ob du diese Schriftart mit anbietest.(durch Einbettung) z.B. so https://dev.to/ziqinyeow/how-to-add-custom-font-file-to-your-react-app-31kb. Google mal nach fonts in react einbinden. Ist derzeit aber nicht wichtig. Hauptsache das Gerüst steht.